corehq/apps/hqwebapp/static/hqwebapp/js/crud_paginated_list.js

```diff
@@ -155,6 +155,9 @@ hqDefine("hqwebapp/js/crud_paginated_list", [
                     statusCode: self.handleStatusCode,
                     success: function (data) {
                         $("#create-item-form")[0].reset();
+                        var button = $("#create-item-form").find(".disable-on-submit");
```

Comment:
ideas to improve this:

a) you can use `$createForm` to avoid another jquery initialization and to be sure you're always modifying the same form.
b) `disable-on-submit` seems fragile in case we want to change the type of button that this form uses and it would make it harder to grep for all the JS using that button. Should add a more semantic CSS class to find this.
c) Fix this for all forms so that they are fixed for all instances of this type of button across HQ. Not sure if that's possible. current code is [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/hq.helpers.js#L26)
d)  Change this button to be an instance of `SaveButton` which I think would handle this case.

Also I think this button is still broken if the request fails. Not sure if we care about that instance though.

c & d may be a bit out of scope for interrupt but a & b seem pretty quick wins though

Reply:
a > good call
b > isn't this the semantic class because of the current code you link in c? I did it this way since it would be broken for all crudpaginated forms that use this
c > I'll timebox looking for a fix for that
d > same as c

Reply:
I think that that code is likely intended for buttons that come up in a modal and then disappear once clicked. 

By adding a semantic class I meant to add it only for this instance of this. So adding the css class to the form in `accounting/forms.py` and referencing it here instead of using the `.disable-on-submit` button which likely isn't intended for this use case